service/history/timerQueueAckMgr.go

```diff
@@ -143,10 +139,9 @@ func newTimerQueueFailoverAckMgr(shard ShardContext, metricsClient metrics.Clien
 		timerQueueShutdown:  timerQueueShutdown,
 		outstandingTasks:    make(map[TimerSequenceID]bool),
 		ackLevel:            ackLevel,
-		readLevel:           ackLevel,
 		minQueryLevel:       ackLevel.VisibilityTimestamp,
 		pageToken:           nil,
-		maxQueryLevel:       maxLevel,
+		maxQueryLevel:       ackLevel.VisibilityTimestamp,
```

Comment:
the max level for failover must be passed in

Reply:
changed